ui/src/geoviite-design-lib/form-layout/form-layout.scss

```diff
@@ -10,6 +10,10 @@
         grid-template-columns: 1fr 1fr;
         grid-gap: 24px;
     }
+
+    &--error-at-bottom {
+        padding-bottom: 0;
+    }
```

Comment:
Tämä liittyy demoamiini dialogin alareunan virheilmoitusbokseihin. Näitä ei lopulta päädytty käyttämään vielä tässä vaiheessa, mutta GK-kaistojen autokonversion myötä saattaa olla mahdollista huomioida virhetilanteita esim. koordinaatin asettamisessa käsin. Toteutetaan ne kuitenkin vasta myöhemmin.